### PR TITLE
[TASK] Improve typesetting examples in rendertests

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

**Review by garvinhicking — APPROVED** [review]

File: Documentation-rendertest/Typesetting/Index.rst [hunk 1]
```diff
@@ -1,170 +1,132 @@
...
+Code
+====
 
-Typesetting H3
---------------
+Inline code
+-----------
 
-Just a line.
+You can highlight inline code using backticks: `some code`, :php:`$somePHP`.
```
Comment (brotkrueml) [inline Documentation-rendertest/Typesetting/Index.rst:65]: Please also add double backticks, like described here: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup

File: Documentation-rendertest/Typesetting/Index.rst [hunk 2]
```diff
@@ -1,170 +1,132 @@
...
+Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
+tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.
+
+Level 4 Header
 ~~~~~~~~~~~~~~
 
-Typesetting H5
-++++++++++++++
+At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd
+gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
```
Comment (brotkrueml) [inline Documentation-rendertest/Typesetting/Index.rst:48]: Please also define a fifth level. I remember, somewhere in the docs I saw that deep level.

File: Documentation-rendertest/Typesetting/Index.rst [hunk 3]
```diff
@@ -1,170 +1,132 @@
...
-~~~~~~~~~~~~~~
+Code blocks
+-----------
 
-Just a line.
+For displaying larger code snippets, use code blocks:
 
-Typesetting H5
-++++++++++++++
+.. code-block:: php
```
Comment (brotkrueml) [inline Documentation-rendertest/Typesetting/Index.rst:72]:
```suggestion
..  code-block:: php
```

File: Documentation-rendertest/Typesetting/Index.rst [hunk 4]
```diff
@@ -1,170 +1,132 @@
...
 
-- three
+*   Item 3
 
-Typesetting requires the prior process of designing a
-font (which is widely but erroneously confused with and substituted for
-typeface). One significant effect of typesetting was that authorship of
-works could be spotted more easily; making it difficult for copiers who
-have not gained permission.
+Ordered List:
```
Comment (brotkrueml) [inline Documentation-rendertest/Typesetting/Index.rst:96]: I would like to see also an example for

    #. First item
    #. Second item
    #. Third item

Personally, I prefer those as I don't have to renumber them when adding a new item somewhere between.

File: Documentation-rendertest/Typesetting/Index.rst [hunk 5]
```diff
@@ -1,170 +1,132 @@
...
-**ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz**
+    This is a blockquote.
+    It can span multiple lines.
 
+Tables
+======
 
-Typesetting H3
---------------
+Tables are represented using pipes and dashes:
```
Comment (brotkrueml) [inline Documentation-rendertest/Typesetting/Index.rst:114]: What about CSV tables?

Comment (linawolf) [reply Documentation-rendertest/Typesetting/Index.rst:114]: There are extended table examples. This should just give a quick overview for those who work on the styles

File: Documentation-rendertest/Typesetting/Index.rst (hunk 1 above)
Comment (linawolf) [reply Documentation-rendertest/Typesetting/Index.rst:65]: The two are actually transformed into the same node and rendered the same. There are pages that test different inline roles more extended

File: Documentation-rendertest/Typesetting/Index.rst (hunk 5 above)
Comment (brotkrueml) [reply Documentation-rendertest/Typesetting/Index.rst:126]: Please add it to the introduction of the page, that it is just a quick overview.

Comment (linawolf) [reply Documentation-rendertest/Typesetting/Index.rst:135]: done, please have another look

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**